blog/_posts/2018-11-09-dvteam-farewell

```diff
@@ -1,51 +1,45 @@
 ---
-title: DV Team Farewell
```

Comment:
could you put this back? you still need the title :)

Reply:
done!